README.md

```diff
@@ -67,46 +68,17 @@ cd build && ctest
 
 NOTE: If you are using cmake versions < 3.24 add `-DCUDA_ARCH=<target_cumpute_arch>` to the command `cmake -S . -B build`
 
-### Rust Bindings
-
-For convenience, we also provide rust bindings to the ICICLE library for the following primitives:
-
-- MSM
-- NTT
-    - Forward NTT
-    - Inverse NTT
-- ECNTT
-    - Forward ECNTT
-    - Inverse NTT
-- Scalar Vector Multiplication
-- Point Vector Multiplication
-
-A custom [build script][B_SCRIPT] is used to compile and link the ICICLE library. The environement variable `ARCH_TYPE` is used to determine which GPU type the library should be compiled for and it defaults to `native` when it is not set allowing the compiler to detect the installed GPU type.
-
-> NOTE: A GPU must be detectable and therefore installed if the `ARCH_TYPE` is not set.
-
-Once you have your parameters set, run:
-
-```sh
-cargo build --release
-```
-
-You'll find a release ready library at `target/release/libicicle_utils.rlib`.
-
-To benchmark and test the functionality available in RUST, run:
-
-```
-cargo bench
-cargo test -- --test-threads=1
-```
+### Bindings
+We support Golang and Rust bindings.
 
-The flag `--test-threads=1` is needed because currently some tests might interfere with one another inside the GPU.
+We offer extensive documentation on them, [Golang bindings][GOLANG_DOCS] and [Rust bindings][RUST_DOCS].
```

Comment:
We support Golang and Rust bindings; their documentation on how to build and use them can be found [here][GOLANG_DOCS] and [here][RUST_DOCS] respectively.